src/core/code-generator.ts

```diff
@@ -82,7 +82,8 @@ export class CodeGenerator {
             scalar: GraphQLID
           }
         ],
-        resolvers: this.options.resolversPath
+        resolvers: this.options.resolversPath,
+        validate: false // Generated input types do not include `class-validator` decorators
```

Comment:
Ok, now that I'm looking at this more closely, I see that some folks might want to validate their `InputType`s.  When you want to go off the beaten path, you'd create your own inputs like this:  https://github.com/goldcaddy77/warthog/blob/193a858f7c038cc95d7965e45cbca53bd20506c7/examples/07-feature-flags/src/feature-flag/feature-flag.resolver.ts#L31

So another way we can do this is add a new config option called `VALIDATE_RESOLVERS` and default it to false.  you can find the different places you'd need to touch here:  

https://github.com/goldcaddy77/warthog/search?q=RESOLVERS_PATH+OR+WARTHOG_RESOLVERS_PATH&unscoped_q=RESOLVERS_PATH+OR+WARTHOG_RESOLVERS_PATH

What do you think?

Reply:
Makes sense! Done in 36f3e2c.